Gododir/gen.go

```diff
@@ -563,26 +273,29 @@ func generateTasks(p *do.Project) {
 }
 
 func generate(service *service) {
-	codegen := "Go"
+	codegen := "--go"
 	if testGen {
-		codegen = "Go.TestGen"
+		codegen = "--go.testgen"
 		service.Fullname = strings.Join([]string{service.Fullname, testsSubDir}, string(os.PathSeparator))
 		service.Output = filepath.Join(service.Output, testsSubDir)
 	}
 
-	fmt.Printf("Generating %s...\n\n", service.Fullname)
-
-	delete(service)
+	fmt.Printf("Working on %s...\n\n", service.Fullname)
+	fmt.Printf("Generating on %s...\n\n", service.Fullname)
 
 	execCommand := "autorest"
 	commandArgs := []string{
-		"-Input", filepath.Join(swaggersDir, "azure-rest-api-specs", service.Input+"."+string(service.Extension)),
-		"-CodeGenerator", codegen,
-		"-Header", "MICROSOFT_APACHE",
-		"-Namespace", service.Name,
-		"-OutputDirectory", service.Output,
-		"-Modeler", string(service.Modeler),
-		"-PackageVersion", sdkVersion,
+		fmt.Sprintf("https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/%s", service.Input),
```

Comment:
on the SDK for .Net repo it now records the commit ID and everything necessary to repro the generation: https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/_metadata (only SQL seems to have updated its SDK since then). See the `readme.md` in that folder for further detail 😉